src/Pretzel.Logic/Configuration.cs

```diff
@@ -55,6 +59,7 @@ private void EnsureDefaults()
             _defaultsConfiguration = new DefaultsConfiguration(_config);
         }
 
+        [OnImportsSatisfied]
         internal void ReadFromFile()
```

Comment:
Does `OnImportSatisfied` need to be on internal method?
Otherwise, it could be private to avoid any unwanted call.

Reply:
As stated earlier from the the [docs](https://docs.microsoft.com/en-us/dotnet/api/system.composition.onimportssatisfiedattribute?view=dotnet-plat-ext-2.1#remarks) it must be public or internal.

> The method that this attribute is applied to must be void, parameterless, and either public or internal.